<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modbus</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>360</y>
     <width>756</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_1">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Disable Display</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>691</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="portSelectBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
